pos_product_packaging_container_deposit/models/pos_order_line.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PosOrderLine(models.Model):
+    _inherit = "pos.order.line"
+
+    is_container_deposit = fields.Boolean()
```

Comment:
The field is already present in `container.deposit.order.line.mixin`

Reply:
Indeed. Inherit mixin just to have the field `is_container_deposit` it's not worth it